### PR TITLE
update for rate-limiting

### PR DESCRIPTION
**Pull request by bdecoste**

### CODE REVIEW

File: instructions/rate-limiting.adoc [hunk 1]
```diff
@@ -1,98 +1,203 @@
...
+====
+
+[#ratelimiting]
+== Rate Limiting
+
+IMPORTANT: The Rate Limiting rules take some time to be applied and reflected. Be patient here!
+
+Here we will limit the number of concurrent requests into recommendation v2
+
+Now apply the rate limiting. To see how this is done,
```
Comment (brian-avery) [inline instructions/rate-limiting.adoc:35]: nit: Style is different between the way info is presented here and fault-injection

File: instructions/rate-limiting.adoc [hunk 2]
```diff
@@ -1,98 +1,203 @@
...
+spec:
+  # quota only applies if you are not logged in.
+  # match: match(request.headers["cookie"], "user=*") == false
+  actions:
+  - handler: handler.memquota
+    instances:
+    - requestcount.quota
+----
+
+Note that the memquota resource `handler` defaines a rate limit of 4 requests per 5 seconds for the `recommendation`
```
Comment (brian-avery) [inline instructions/rate-limiting.adoc:128]: defines

File: instructions/rate-limiting.adoc [hunk 3]
```diff
@@ -1,98 +1,203 @@
...
+destination. Next, deploy the rate limiting resources.
+
+[source,bash,subs="+macros,+attributes"]
+----
+$ oc -n istio-tutorial create -f http://github.com/thoraxe/istio-lab-summit-2019/blob/master/src/istiofiles/recommendation_rate_limit.yml
```
Comment (brian-avery) [inline instructions/rate-limiting.adoc:133]: oc -n istio-tutorial create -f src/istiofiles/recommendation_rate_limit.yml
destinationrule.networking.istio.io/recommendation created
the namespace from the provided object "istio-system" does not match the namespace "istio-tutorial". You must pass '--namespace=istio-system' to perform this operation.
the namespace from the provided object "istio-system" does not match the namespace "istio-tutorial". You must pass '--namespace=istio-system' to perform this operation.
the namespace from the provided object "istio-system" does not match the namespace "istio-tutorial". You must pass '--namespace=istio-system' to perform this operation.
the namespace from the provided object "istio-system" does not match the namespace "istio-tutorial". You must pass '--namespace=istio-system' to perform this operation.
the namespace from the provided object "istio-system" does not match the namespace "istio-tutorial". You must pass '--namespace=istio-system' to perform this operation.

File: instructions/rate-limiting.adoc [hunk 4]
```diff
@@ -1,98 +1,203 @@
...
+  namespace: istio-system
+spec:
+  quotas:
+  - name: requestcount.quota.istio-system
+    maxAmount: 500
+    validDuration: 1s
+    # The first matching override is applied.
+    # A requestcount instance is checked against override dimensions.
+    overrides:
+    # The following override applies to 'reviews' regardless
```
Comment (brian-avery) [inline instructions/rate-limiting.adoc:71]: recommendation

**Comment by thoraxe** [comment]
Is this ready?

**Comment by brian-avery** [comment]
For me, at least, even after the requested changes, this is not working.

File: instructions/rate-limiting.adoc (hunk 3 above)
Comment (knrc) [reply instructions/rate-limiting.adoc:133]: The yaml file appears to be installing into two namespaces so try and run it without the '-n' option

**Pull request merged**